scene/main/node.cpp

```diff
@@ -695,6 +696,16 @@ void Node::_propagate_pause_notification(bool p_enable) {
 	data.blocked--;
 }
 
+void Node::_propagate_suspend_notification(bool p_enable) {
+	notification(p_enable ? NOTIFICATION_SUSPENDED : NOTIFICATION_UNSUSPENDED);
+
+	data.blocked++;
+	for (KeyValue<StringName, Node *> &K : data.children) {
```

Comment:
```suggestion
	for (KeyValue<StringName, Node *> &KV : data.children) {
```
Fresh guideline from Godot Sprint. (use KV for KeyValue)